llvm/unittests/Support/CommandLineTest.cpp

```diff
@@ -2206,4 +2206,25 @@ TEST(CommandLineTest, DefaultValue) {
   EXPECT_EQ(1, StrInitOption.getNumOccurrences());
 }
 
+TEST(CommandLineTest, UnknownSubcommand) {
```

Comment:
I might be tempted to name this `UnknownCommands` so that it covers top-level and subcommands, since it doesn't look like there's checking for the unknown command message elsewhere, at a quick glance.